server/index.js

```diff
@@ -4,33 +4,48 @@ const app = express();
 const port = 3000;
 const path = require('path');
 const bodyParser = require('body-parser');
-const db = require('../database/index.js');
+
+const mongo = require('../database/mongo.index.js')
 
 app.use(bodyParser.json());
 
 
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use('/books/:id', express.static(path.join(__dirname, '../public')));
 
-app.get('/books/:id/authors/title', async(req, res) => {
-  console.log('server/index line 16')
-  const title = await db.getBook(req.params.id);
-  res.json(title);
+//MONGO VERSION
+
+app.get('/books/:id/authors/title', async (req, res) => {
+    console.log('server/index line 16')
+  var booksId = req.params.id
+  var book =  await mongo.getBooks(booksId)
+  res.status(200).json(book)
+
 });
 
 app.get('/books/:id/authors/:id', async (req, res) => {
   console.log('server/index line 21')
-  const author = await db.getAuthor(req.params.id);
+  var authorId = req.params.id
+  const author = await mongo.getAuthor(authorId);
   res.json(author);
 });
 
+
+
 app.get('/books/:id/authors/:id/titles', async (req, res) => {
-  const books = await db.getAuthorTitles(req.params.id);
+  var authorId = req.params.id
+  const books = await mongo.getAuthorTitles
+
+  (authorId);
   res.json(books);
 });
 
+
 app.post('/books/:id/authors/status', async (req, res) => {
-  const status = await db.updateStatus(req.body.status, req.body.id);
+  var bookStatus = req.body.status;
+  var booksId = req.body.id
+  const status = await mongo.updateStatus
+  (bookStatus, booksId);
```

Comment:
Is this part of the function invocation on line 47? Why not just keep it on the same line?